### ISSUE TITLE
split delivery

### ISSUE THREAD

**Issue by Ashember** [Iku/Google-Forms-to-Discord#86]
is there a way to split the delivery to discord if it goes over 2000 letters.

**Comment by ItsKelo** [comment]
I think the code automatically extends the embed to include a ____ Cont. section. Is this what you mean?

**Comment by Ashember** [comment]
yeah but I dont' think it does or I didn't set it up right because I constantly get the 2000 letter error
,

**Comment by udit2303** [comment]
I don't believe so, since it would send the embed only once so you could try to create 2 forms or more for this.
